be/src/util/timezone_utils.cpp

```diff
@@ -42,7 +42,65 @@ static phmap::flat_hash_map<std::pair<std::string_view, std::string_view>, int64
 void TimezoneUtils::init_time_zones() {
     // timezone cache list
     // We cannot add a time zone to the cache that contains both daylight saving time and winter time
-    static std::vector<std::string> timezones = {"UTC", "Asia/Shanghai", "Asia/Hong_Kong"};
+    static std::vector<std::string> timezones = {
+        "Africa/Abidjan","Africa/Accra","Africa/Addis_Ababa","Africa/Algiers","Africa/Asmara",
```

Comment:
better we can move this to another file and include that file here.
Like create file "timezone.dat" and the file content like this
```
"Africa/Abidjan",
"Africa/Accra"
```
Then these code can be changed 
```
static std::vector<std::string> timezones = {
#include "timezone.dat"
}
```

Reply:
ok, good idea